src/js/pages/Topic.js

```diff
@@ -2,41 +2,50 @@ import React from 'react';
 import { withRouteData } from 'react-static';
 import { get } from 'lodash';
 import { cleanServiceLinks } from 'js/helpers/cleanData';
+import PropTypes from 'prop-types';
 
 // TODO: this jsonFileData is temporary. Add it to Wagtail API
 import jsonFileData from '__tmpdata/pages';
-import Hero from 'js/modules/Hero';
 import TileGroup from 'js/modules/TileGroup';
+import PageBreadcrumbs from 'js/modules/PageBreadcrumbs';
+import PageHeader from 'js/modules/PageHeader';
+import SectionHeader from 'js/modules/SectionHeader';
 import FormFeedback from 'js/page_sections/FormFeedback';
 import ThreeOneOne from 'js/page_sections/ThreeOneOne';
 
 const Topic = ({ topic }) => {
   const title = get(topic, "text", null);
-  const body = get(topic, "description", null);
+  const callToAction = get(topic, "callToAction", null);
+  const description = get(topic, "description", null);
   const links = get(topic, "services", null);
+  const theme = get(topic, "theme", null);
   const relatedLinks = cleanServiceLinks(links);
   const services311 = get(jsonFileData, "services311", null);
 
   return (
     <div>
-      <Hero callout={title} />
+      <PageBreadcrumbs title={title} order={['topic']} topic={theme} />
```

Comment:
any reason your changing the name from theme to topic?
<PageBreadcrumbs title={title} order={['theme']} theme={theme} />

Reply:
Breadcrumbs needs more refactoring as we discussed the ordering thing is clever, but could be improved by passing in an array of object where the keys are the classname and the value is the object in the order props. Also, the css is pretty coupled to the one use case in the Services page. We could try to make the classnames abstracted away form topics, themes, titles and use something like child, parent, grandparent. Or we could be clever with nth and last-child psuedo- selectors. 😓